scripts/process.py

```diff
@@ -0,0 +1,111 @@
+    'Netherlands': {'daily': '', 'monthly': 'EXNEUS', 'yearly': ''},
+    'New Zealand': {'daily': 'DEXUSNZ', 'monthly': 'EXUSNZ', 'yearly': 'AEXUSNZ'},
+    'Norway': {'daily': 'DEXNOUS', 'monthly': 'EXNOUS', 'yearly': 'AEXNOUS'},
+    'Portugal': {'daily': '', 'monthly': 'EXPOUS', 'yearly': ''},
+    'Singapore': {'daily': 'DEXSIUS', 'monthly': 'EXSIUS', 'yearly': 'AEXSIUS'},
+    'South Africa': {'daily': 'DEXSFUS', 'monthly': 'EXSFUS', 'yearly': 'AEXSFUS'},
+    'South Korea': {'daily': 'DEXKOUS', 'monthly': 'EXKOUS', 'yearly': 'AEXKOUS'},
+    'Spain': {'daily': '', 'monthly': 'EXSPUS', 'yearly': ''},
+    'Sri Lanka': {'daily': '', 'monthly': 'EXSLUS', 'yearly': ''},
+    'Sweden': {'daily': 'DEXSDUS', 'monthly': 'EXSDUS', 'yearly': 'AEXSDUS'},
+    'Switzerland': {'daily': 'DEXSZUS', 'monthly': 'EXSZUS', 'yearly': 'AEXSZUS'},
+    'Taiwan': {'daily': 'DEXTAUS', 'monthly': 'EXTAUS', 'yearly': 'AEXTAUS'},
+    'Thailand': {'daily': 'DEXTHUS', 'monthly': 'EXTHUS', 'yearly': 'AEXTHUS'},
+    'United Kingdom': {'daily': 'DEXUSUK', 'monthly': 'EXUSUK', 'yearly': ''},
+    'Venezuela': {'daily': 'DEXVZUS', 'monthly': 'EXVZUS', 'yearly': 'AEXVZUS'}
+}
+
+
+def rename(package: PackageWrapper):
+    package.pkg.descriptor['resources'][0]['name'] = 'exchange-rates-daily'
+    package.pkg.descriptor['resources'][0]['path'] = 'data/daily.csv'
+    package.pkg.descriptor['resources'][1]['name'] = 'exchange-rates-monthly'
+    package.pkg.descriptor['resources'][1]['path'] = 'data/monthly.csv'
+    package.pkg.descriptor['resources'][2]['name'] = 'exchange-rates-yearly'
+    package.pkg.descriptor['resources'][2]['path'] = 'data/yearly.csv'
+    yield package.pkg
+    res_iter = iter(package)
+    first: ResourceWrapper = next(res_iter)
+    second: ResourceWrapper = next(res_iter)
+    third: ResourceWrapper = next(res_iter)
+    yield first.it
+    yield second.it
+    yield third.it
+    yield from package
+
+
+def extract_exchange_rates(frequency):
+    for country in country_codes:
+        country_frequency_ticker = country_codes[country][frequency]
+        url = base_url + country_frequency_ticker + '.txt'
+        if country_frequency_ticker:
+            response = urllib.request.urlopen(url)
+            lines = response.readlines()
+            header = True
+            for line in lines:
+                line_str = line.decode('utf-8')
+                if not header:
+                    date_value = line_str.split(' ')
+                    date = date_value[0]
+                    value = date_value[1:]
+                    value = "".join(value).strip()
+                    if value == '.':
+                        value = ''
+                    if country in inverse:
+                        if value != '':
+                            value = str(round(1 / float(value), 4))
+
+                    yield {
+                        'Date': date,
+                        'Country': country,
+                        'Value': value
```

Comment:
Think `Rate` or  `Exchange Rate` would be better name than `Value`

Reply:
FIXED